### DISCUSSION TITLE
Touch swipe to scroll not working

### DISCUSSION THREAD

**Discussion by clibu** [FRSOURCE/tiny-carousel#220] [Q&A] [answered]
Touch swipe works fine in the Doc demo's however it isn't working in my app. Kbd left/right arrow do work. No doubt either some css or JS code of mine is causing this, but so far I haven't been able to track this down.

I have ``<meta name="viewport" content="width=device-width, minimum-scale=1">`` in my ``<head>`` but don't think that is the cause.

Any suggestions of things to check?

**Comment by clibu** [comment, marked as answer]
Ok, I've more less tracked it down. I was overriding ``overflow: scroll`` on the carousel element. I have a few more gray hairs than before.😀

A basic description of the CSS usage and requirements would be a welcome addition to the docs.

**Reply by FRSgit** [reply]
Great to hear that! I've [added a tip about your issue](https://www.frsource.org/tiny-carousel/guide/usage/#core), do you think it's enough?

![image](https://user-images.githubusercontent.com/10456649/147997022-d4d8f815-3463-4a07-9892-b4d4629adead.png)

**Reply by clibu** [reply]
@FRSgit  Yes that will help, thanks.
